compiler/nim.nim

```diff
@@ -80,14 +80,25 @@ proc handleCmdLine(cache: IdentCache; conf: ConfigRef) =
     else:
       conf.projectPath = canonicalizePath(conf, getCurrentDir())
     loadConfigs(DefaultConfig, cache, conf) # load all config files
-    let scriptFile = conf.projectFull.changeFileExt("nims")
-    if fileExists(scriptFile):
-      runNimScript(cache, scriptFile, freshDefines=false, conf)
+
+    proc runNimScriptIfExists(scriptFile:string)=
```

Comment:
Space after the colon: Just like in written English.

Reply:
done